src/Uno.UI/UI/Xaml/Controls/Flyout/FlyoutPopupPanel.cs

```diff
@@ -25,6 +25,16 @@ public FlyoutBasePopupPanel(FlyoutBase flyout) : base(flyout._popup)
 			_flyout = flyout;
 		}
 
+		protected override Size MeasureOverride(Size availableSize)
+		{
+			if (this._flyout.Placement == FlyoutPlacementMode.Full)
+			{
+				return base.MeasureOverride(new Size(double.MaxValue, double.MaxValue));
```

Comment:
availableSize typically takes on the value of the flyout, without stretching (so, only taking the size of the content and not covering the entire screen with semi-opacity).

I used MaxValue because it worked and I didn't see a specific need to constrain the size. Is there an upside to using the window size?

Reply:
We should use whichever's correct, that's why I'm asking. :)

Naively I thought that UWP would wrap content to the window, but on a quick test, it seems to actually put it in a `ScrollViewer` and give it as much size as it wants. So maybe this is the right approach. 

My test is below if you want to verify it with your changes. Seems like Uno master actually wraps the content, unlike UWP. If your changes affect this, it should be listed under Breaking Changes.

```xaml
		<Button Content="Show flyout" >
			
			<Button.Flyout>
				<Flyout Placement="Full">
					<Border Margin="100"
							Background="LightPink">
						<TextBlock Text="umwea thueas uhteastuh eaot hueas huestah uetas huetash ueas htueasot aeso htueasuho etaosh sae htuas thueas theas nutheas nuhtae sntheua ntehua teash esa hteasu nheausth euas hteasuh taesth ueasth ueas heauo nhsa sntehao suteh"
								   TextWrapping="Wrap" />
					</Border>
				</Flyout>
			</Button.Flyout>
		</Button>
```